src/datastore/pg-write-store.ts

```diff
@@ -2450,6 +2449,16 @@ export class PgWriteStore extends PgStore {
     if (this.isEventReplay && skipDuringEventReplay) {
       return;
     }
+    if (process.env.IBD_MODE_UNTIL_BLOCK) {
+      const db = new PgWriteStore(sql);
+      const chainTip = await db.getChainTip(sql);
```

Comment:
You can use `this` instead of having to instantiate a new `PgWriteStore`

Reply:
done